test/framework/rootpod_executor.go

```diff
@@ -74,7 +74,12 @@ func (e *rootPodExecutor) Execute(ctx context.Context, command string) ([]byte,
 	command = fmt.Sprintf("chroot /hostroot %s", command)
 	reader, err := e.executor.Execute(ctx, e.Pod.Namespace, e.Pod.Name, e.Pod.Spec.Containers[0].Name, command)
 	if err != nil {
-		return nil, err
+		var response []byte
+		if reader != nil {
+			response, _ = io.ReadAll(reader)
```

Comment:
You could add a doc string explaining why we ignore the error:

```suggestion
			// We try to read the response on best effor and ignore the returned error from the response read operation to do not mask the returned command error.
			response, _ = io.ReadAll(reader)
```

Reply:
Why don't we use https://pkg.go.dev/errors#Join ? I think generally in go skipping errors is not a good practice and linters may fail https://golangci-lint.run/usage/linters/#errcheck